ai-server/app/ml/generate_training_data.py

```
@@ -0,0 +1,153 @@
+향상된 알고리즘으로 ML 훈련 데이터 생성
+"""
+
+import sys
+import os
+
+# 프로젝트 루트를 Python 경로에 추가
+current_dir = os.path.dirname(os.path.abspath(__file__))
+ai_server_dir = os.path.dirname(os.path.dirname(current_dir))
+sys.path.insert(0, ai_server_dir)
+
+from sqlalchemy import create_engine
```

Ruff (0.12.2)

`sqlalchemy.create_engine` imported but unused

Remove unused import: `sqlalchemy.create_engine`

(F401)